src/Pages/Homepage.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import fbIcon from '../static/images/icons8-facebook-48.png';
+import twitterIcon from '../static/images/icons8-twitter-48.png';
+import '../static/css/homepage.css';
+
+
+function Homepage() {
+  return (
+    <>
+      <header>
+        <div className="asd">
+          <h1 className="title">The Template Project</h1>
```

Comment:
generally a good idea to avoid naming classes after html tags. Does this element need a class, or would it be better to applied to the style onto the `h1` element?

Reply:
I agree that the classes are unnecessary class name; now removed